indexd/index/drivers/alchemy.py

```diff
@@ -633,6 +635,71 @@ def add(self,
 
             return record.did, record.rev, record.baseid
 
+    def add_blank_record(self, did=None, uploader=None, baseid=None):
```

Comment:
also `did` should not be provided for this case, indexd should generate a new GUID and send it to fence

Reply:
Fixed in 83f5b6b6dfc04b9eaa7d6502d320ae6c639ba1aa